bindata/network/frr-k8s/frr-k8s.yaml

```diff
@@ -86,7 +86,7 @@ spec:
         - --node-name=$(NODE_NAME)
         - --namespace=$(NAMESPACE)
         - --metrics-bind-address=127.0.0.1:7572
-        - --log-level=info
+        - $(LOG_LEVEL)
```

Comment:
I forgot to ask but I am assuming the default log level is `info` and it's no problem that this ends up being an empty array element?

Reply:
yep, totally valid question. Default is `info`, and I tried locally :-)